app/src/main/java/edu/uc/hickmadc/customnotifications/adapters/GroupsAdapter.kt

```diff
@@ -23,23 +26,21 @@ class GroupsAdapter(private val dataSet: ArrayList<Group>) :
         )
     }
 
-    // Replace the contents of a view (invoked by the layout manager)
+    /**
+     * Replace the contents of a view (invoked by the layout manager)
+     * then gets a group from dataSet and bind it to the empty viewHolder
```

Comment:
Changed this from two lines to a single sentence, it looks a little weird in AS when you hover if its not like this.